README.md

```
@@ -3,18 +3,37 @@
-- [Tureng](https://tureng.com/tr/turkce-ingilizce) - Comprehensive Turkish-English Dictionary
-- [Dictionary.com](https://www.dictionary.com/) - English Dictionary with Definitions
-- [Longman Dictionary](https://www.ldoceonline.com/) - Learner's Dictionary of Contemporary English
-- [TR-EX](https://tr-ex.me/) - Context Examples and Usage
+
+- [Apertium](https://en.wikipedia.org/wiki/Apertium) – A rule-based, open-source machine translation platform supporting multiple language pairs.
+- [Dictionary.com](https://www.dictionary.com/) – English dictionary with definitions.
+- [FreeDict](https://freedict.org/) – A comprehensive source of truly free bilingual dictionaries in over 45 languages.
+- [Longman Dictionary](https://www.ldoceonline.com/) – A learner-friendly dictionary of contemporary English.
+- [TR-EX](https://tr-ex.me/) – Contextual sentence examples and usage suggestions.
+- [Tureng](https://tureng.com/tr/turkce-ingilizce) – Comprehensive Turkish-English dictionary.
+- [Wiktionary](https://en.wikipedia.org/wiki/Wiktionary) – A user-generated, multilingual, and open-content dictionary with definitions, pronunciations, etymologies, and translations. ([GitHub][1], [FreeDict][2])
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "1"

(MD052, reference-links-images)